Backend/.gitignore

```diff
@@ -36,6 +36,8 @@ s3
 /dev/osrm/*
 !/dev/osrm/README
 !/dev/osrm/Dockerfile.osrm
+stack.yaml*
```

Comment:
remove this.

Reply:
> remove this.

Is this a problem? if anyone want to use stack for some reason, why can't they?